tests/cases/fourslash/getEmitOutputTsxFile_ReactNative.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path="fourslash.ts" />

```

Comment:
This shouldn't be a fourslash test (they are much slower than compiler baselines). See other tests in `tests/cases/compiler` for how to write a multi-file baseline test (search "`@filename`")